src/components/dayCard/DayCard.test.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen } from '@testing-library/react';
+import DayCard from './DayCard';
+import Wind from '../../assets/icons/wind.svg';
+
+describe('DayCard', () => {
+  test('it renders the day of the week', () => {
+    render(<DayCard day='Saturday' />);
+    const dayOfTheWeek = screen.getByText('Saturday');
+    expect(dayOfTheWeek).toBeInTheDocument();
+  });
+
+  test('it renders the date', () => {
+    render(<DayCard day='Saturday' date='March 2nd' />);
+    const dayOfTheWeek = screen.getByText('March 2nd');
+    expect(dayOfTheWeek).toBeInTheDocument();
+  });
+
+  test('it renders the correct icon', () => {
```

Comment:
Here we have the word `correct`, we don't have on all the tests. Shall we unify it?